src/x11.cc

```diff
@@ -337,6 +337,8 @@ conky::range_config_setting<int> border_width("border_width", 0,
 conky::simple_config_setting<bool> use_xft("use_xft", false, false);
 #endif
 
+conky::simple_config_setting<bool> forced_redraw("forced_redraw", false, false);
```

Comment:
Set default value to `true` to keep existing behaviour.

Reply:
Actually, I wanted to remove these lines of code completely because they are unnecessary for the normal use. I think, we should keep them `false` because it'll work well on all WMs by default .

Reply:
For those who wants to refresh Conky on desktop change will set it to `true`.

Reply:
>Actually, I wanted to remove these lines of code completely because they are unnecessary for normal use. I think, we should keep them `false` because it'll work well on all WMs by default.

@brndnmtthws Thoughts on doing this first? See if we don't have to expose a config.

Reply:
I'm leaning toward the idea of convention over configuration. If there is a correct default, we should just stick with that.

Conky is sometimes too configurable :) But at the same time, I often changes things and later discover someone was depending on some strange esoteric behaviour that Conky had, and it breaks backward compatibility.

If you feel strongly this should just be the default, then sure let's do it.

Reply:
I think the issue is that we have many configuration settings and that it is probable that some of them does not work correctly. The other issue is that we need a way to test the said config in the said environment because we have users with issues that only can occur on `lxde`, `xfce4`, major bug on `bspwm`, and others. Virtual machines or something.

If this impact somebody, but worked okay in the past, we can bisect and find this commit. Do you have any good reasons/examples why we should make a config instead? This config seems trivial, but is it correct? Maybe. We have some open issue(s) caused by users switching workspaces #718. Is it a bugfix too?

Reply:
We can say this fixes #512 and worry about a config later.

Reply:
Is there any reason

> One last thing: can you reformat as per: https://gitlab.com/brndnmtthws-oss/conky/-/jobs/167347013
> 
> If you have `clang-format` installed you can just run `make clang-format`.

Done.